components/drag-and-drop/index.tsx

```diff
@@ -1,46 +1,364 @@
+									PNG or SVG
+								</Typography>
+							</div>
+							<div className={styles.imageInfoItem}>
+								<span>{isSvg || is310px ? <SvgCheck disabled={isSvg} /> : <SvgError />}</span>
+								<Typography variant="regularBody" weight="medium" muted={isSvg}>
+									310px or higher
+								</Typography>
+							</div>
+						</div>
+						<div className={classnames(styles.imageOptionsWrapper, { [styles.dark]: isDark })}>
+							<Typography variant="footer" weight="semiBold" color="gray">
+								Advanced
+							</Typography>
+							<div className={styles.imageOptions}>
+								<div className={styles.imageInfoItem}>
+									<Checkbox name="pwa" id="pwa" disabled={PWADisabled} onChange={() => setPwa(!pwa)}>
+										<Typography variant="regularBody" weight="medium" muted={PWADisabled}>
+											PWA compatible
+										</Typography>
+										<Typography variant="footer" weight="semiBold" color="green" muted={PWADisabled || isSvg}>
+											512px or higher{' '}
+											<SvgCheck className={classnames(styles.pwaCheck, { [styles.hide]: PWADisabled || isSvg })} />
+										</Typography>
+									</Checkbox>
+								</div>
+								<div className={styles.imageInfoItem}>
+									<Checkbox name="dark" id="dark" disabled onChange={() => setDarkMode(!darkMode)}>
+										<Typography variant="regularBody" weight="medium" muted>
+											Dark Mode version
+										</Typography>
+										<Typography variant="footer" weight="semiBold" color="green" muted>
+											Available soon
+										</Typography>
+									</Checkbox>
+								</div>
+							</div>
+						</div>
+						<div className={classnames(styles.imageFooter, { [styles.dark]: isDark })}>
+							<Button variant="transparent" color="gray" onClick={resetImage}>
+								Re-upload
+							</Button>
+							<Button
+								color="white"
+								background="bgLink"
+								className={styles.imageGenerate}
+								onClick={() => generateFavicon(image)}>
+								Generate Favicon
+							</Button>
+						</div>
+					</>
+				)}
+				{zipData && (
+					<>
+						<div>
+							<div className={styles.imagePreviewDownload}>
+								<div className={classnames(styles.preview, styles.circle)}>
+									<img src={imageBase64} alt="Favicon preview circle" />
+								</div>
+								<div className={classnames(styles.preview, styles.square)}>
+									<img src={imageBase64} alt="Favicon preview square" />
+								</div>
+								<div className={classnames(styles.preview, styles.rounded)}>
+									<img src={imageBase64} alt="Favicon preview rounded" />
+								</div>
+							</div>
+							<Typography variant="title" weight="bold" className={styles.imagePreviewTitle}>
+								Your new favicon is ready!
+							</Typography>
+							<Typography variant="footer" weight="semiBold" color="gray" className={styles.imagePreviewSubtitle}>
+								We’ve prepared a zip file with all the sizes and a README with the code plus other resources.
```

Comment:
Instead of code we can say meta tags here

Reply:
@otsugua 👀